src/modules/fancyzones/editor/FancyZonesEditor/Utils/FancyZonesEditorIO.cs

```diff
@@ -846,6 +880,21 @@ private bool SetTemplateLayouts(List<TemplateLayoutWrapper> templateLayouts)
             return true;
         }
 
+        private bool SetFastAccessKeys(List<FastAccessLayoutKeysWrapper> fastAccessKeys)
+        {
+            if (fastAccessKeys == null)
+            {
+                return false;
+            }
+
+            foreach (var wrapper in fastAccessKeys)
```

Comment:
Perhaps we should clean up `FastAccessKeysModel` here, as this function call should override all keys.

Reply:
In the current implementation, parsing is called just once at startup, but basically, you're right, I'll update the code.